src/pretix/base/models/vouchers.py

```diff
@@ -506,11 +506,16 @@ def budget_used(self):
 
 
 class InvoiceVoucher(LoggedModel):
+    PRICE_MODE_NONE = 'none'
+    PRICE_MODE_SET = 'set'
+    PRICE_MODE_SUBTRACT = 'subtract'
+    PRICE_MODE_PERCENT = 'percent'
+
     PRICE_MODES = (
```

Comment:
Please move this out to an `enum` for Django TextChoice, so that other code can import and use it.